src/Pages/Signup/index.js

```diff
@@ -1,7 +1,258 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import { API } from '../../config';
+import { fetchPost } from '../../utils/fetches';
+import { REGEXP, validate } from '../../utils/regex';
+import './index.scss';
+
+class Signup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      nickname: '',
+      phone_number: '',
+      gender: '',
+      year: '',
+      month: '',
+      day: '',
+    };
+  }
+
+  validateInputData = (id, pw) => {
+    return (
+      validate(id, REGEXP.emailRegExp) && validate(pw, REGEXP.passwordRegExp)
+    );
+  };
+
+  handleInput = (e) => {
+    this.setState({ [e.target.name]: e.target.value });
+    console.log(this.state);
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    if (!this.validateInputData(this.state.email, this.state.password)) return;
+    let response = fetchPost(API, {
```

Comment:
```suggestion
  handleSubmit = (e) => {
    e.preventDefault();
    
    if (!this.validateInputData(this.state.email, this.state.password)) return;
    
    let response = fetchPost(API, {
```
가독성을 위해 구분되는 요소들은 개행으로 구분해주면 좋겠습니다